src/Compatibility/Microsoft.DotNet.PackageValidation/GetLastStablePackage.cs

```diff
@@ -50,7 +50,8 @@ protected override void ExecuteCore()
             
             if (version == null)
             {
-                throw new Exception(string.Format(Resources.BaselinePackageNotFound, PackageId, Environment.NewLine + string.Join(Environment.NewLine + " - ", NugetFeeds)));
+                Log.LogNonSdkWarning(DiagnosticIds.NoBaselinePackageFound, string.Format(Resources.BaselinePackageNotFound, PackageId, Environment.NewLine + string.Join(Environment.NewLine + " - ", NugetFeeds)));
```

Comment:
Should we also treat specially the case that one version was pushed, but it is the same as the one you are trying to build which is why this task wouldn't have picked it? It will be wierd to see a message saying that no package was found in the feeds when the user clearly knows that their 1.0.0 package is there.

Reply:
yeah i will add a log warning if the latest stable version == package version.